packages/untp-playground/src/lib/utils.ts

```diff
@@ -58,3 +58,46 @@ export const downloadJson = (data: Record<string, any>, filename: string) => {
     throw new Error('Data is not JSON-serializable');
   }
 };
+
+/**
+ * Returns the type of the given value as a string.
+ *
+ * This function uses `Object.prototype.toString` to get the internal
+ * [[Class]] property of the value, which is a more reliable way to
+ * determine the type of an object than using `typeof` or `instanceof`.
+ *
+ * The result is a string such as "String", "Number", "Array", "Object", etc.
+ *
+ * @param value - The value whose type is to be determined.
+ * @returns The type of the value as a string.
+ */
```

Comment:
Looks like you deleted the function but left the docstring for it? That might be confusing for someone reading this later... can we remove it too?